cmd/createtree/doc.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package main contains the implementation and entry point for the createtree
+// command.
+//
+// Example usage:
+// $ ./createtree \
+//     --admin_endpoint=host:port \
+//     --pem_key_path=/path/to/pem/file \
+//     --pem_key_password=mypassword
```

Comment:
What advice would you have for users that want to provision a tree with a different HashingStrategy?